RELEASE_NOTES.md

```diff
@@ -19,6 +19,11 @@ v1.0.0-alpha.x
   `OPENASSETIO_ENABLE_PYTHON_TEST_VENV` is enabled.
   [#629](https://github.com/OpenAssetIO/OpenAssetIO/issues/629)
 
+- Changed `--install-folder` location of conan install in bootstrap
+  scripts from `$CONAN_USER_HOME` to `$WORKSPACE/.conan`. Users who
+  have been using the bootstrap scripts directly may need to update
+  their toolchain CMake arguments.
+
 ### New Features
```

Comment:
[consider][minor] Should we add a release note that Python wheels are downloadable through the build artifacts of the "Build wheels" Github Actions workflow?  Might be useful to some who want bleeding edge without building themselves.

Reply:
Yeah, i think that's a good idea.

Reply:
Improvement works for me 👍

Reply:
[61db753](https://github.com/OpenAssetIO/OpenAssetIO/pull/665/commits/61db7533724539e8d8098c751462fc0d18184078)